lib/livebook_web/live/file_select_component.ex

```diff
@@ -560,6 +571,11 @@ defmodule LivebookWeb.FileSelectComponent do
     {:noreply, assign(socket, error: false)}
   end
 
+  def handle_event("set_default_directory", %{"path" => path}, socket) do
```

Comment:
If a different file system is selected we want to account for this too. We should do this:

```elixir
{dir, _prefix} = dir_and_prefix(assigns.file)
Livebook.Settings.set_default_dir(dir)
```

Reply:
fix it. https://github.com/livebook-dev/livebook/pull/2046/commits/1fc1d415a37eedac6369b45e156e816fd2345fc7

But now we have another problem.
when restart livebook, It comes out like this:

<img width="999" alt="image" src="https://github.com/livebook-dev/livebook/assets/43606066/0d873b63-07e9-4de7-9d7c-420a93326394">